src/libraries/System.DirectoryServices/.editorconfig

```diff
@@ -0,0 +1 @@
+dotnet_diagnostic.IDE0060.severity = none
```

Comment:
Why is IDE0060 being disabled for this library?

Reply:
There is a lot to fix (IIRC 300+ places) in this library and I thought we will exclude other analyzers here too. Thoughts?

Reply:
What does IDE0060 have to do with this PR for IDE0059 though?

Reply:
I had more WIP changes in local trees

Reply:
Hmm, there is something fishy going on

Reply:
The files in not needed in this PR